### PR TITLE
fix(website): add alt texts to images - I113

### PR DESCRIPTION
**Pull request by devcer**
# Issue #113 
Added missing alt text to few images

### Changes
Added alt text to two images with missing alt text

### Related Issues
- Issue #113

### CODE REVIEW

**Comment by devcer** [comment]
Hi @irmerk , I made the requested changes.

**Comment by irmerk** [comment]
@devcer sorry I must have missed this! Could you rebase to resolve conflicts and we can merge this?

**Comment by devcer** [comment]
Resolved the conflicts @irmerk ! you can merge this now. :)

File: website/core/Footer.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ class Footer extends React.Component {
 
             <div className="linux-foundation-logo">
               <a href="https://www.linuxfoundation.org/projects/">
-                <img src="/img/LF-logo.png" />
+                <img src="img/LF-logo.png" alt="The Linux Foundation Projects"/>
```
Comment (jeromesimeon) [inline website/core/Footer.js]: This change will break the LF logo again. Please use the `/`. See #92

Comment (devcer) [reply website/core/Footer.js]: Sorry. I didn't see this. edited the commit. please check.

**Pull request merged**